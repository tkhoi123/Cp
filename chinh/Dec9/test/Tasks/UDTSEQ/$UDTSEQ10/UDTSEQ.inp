145 92
51 142 -823
24 142 -372
65 128 516
54 95 -634
4 81 -861
69 69 153
16 45 -556
10 52 -504
98 121 -870
57 101 409
10 52 -856
43 112 341
78 91 -31
16 55 -924
81 125 -789
83 138 -473
81 138 25
101 115 610
37 119 -371
45 50 268
14 17 -608
30 40 -55
43 122 -178
18 87 161
67 79 723
53 139 -506
10 27 968
76 93 984
27 131 -161
79 117 397
34 55 -154
16 118 138
83 123 -846
107 127 -756
3 62 -619
71 107 -139
4 99 924
106 131 175
43 63 -593
72 140 -243
69 79 -591
28 96 991
24 65 444
31 87 784
12 30 47
134 142 644
18 45 144
34 49 221
28 44 265
16 106 -113
24 87 -346
19 55 -661
9 139 -676
125 132 207
95 123 420
12 17 380
32 62 510
57 121 -121
13 93 -26
46 134 -356
42 120 370
58 60 -326
22 131 -803
47 79 94
28 75 -151
37 81 901
1 117 633
28 31 -860
54 142 932
3 60 -9
50 136 650
2 58 879
80 87 32
122 130 252
42 103 474
22 115 294
14 89 592
88 116 700
133 135 86
79 81 140
49 113 -68
73 112 -344
51 90 828
50 98 423
107 141 -223
106 142 131
48 133 -357
53 67 -646
16 51 362
52 140 255
48 130 -235
107 128 217
