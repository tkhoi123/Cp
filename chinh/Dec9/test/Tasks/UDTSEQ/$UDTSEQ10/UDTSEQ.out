633 1512 884 947 947 947 947 947 271 -121 -121 306 280 264 264 -829 -829 -296 -957 -957 -957 -1466 -1466 -1740 -1740 -1740 -1901 -2624 -2624 -2679 -1942 -572 -572 -505 -505 -505 25 25 25 25 80 924 494 494 497 553 647 55 -13 839 576 469 677 975 975 2714 3002 2676 1797 1797 2132 2132 2241 2834 3350 2906 3629 4275 3837 3684 3545 3302 2958 2958 2958 4093 4093 4062 4599 4405 3641 3461 2142 2142 2142 2142 2142 2211 2211 1619 791 822 822 -136 284 918 -73 -943 -1366 -2290 -1680 -2089 -2089 -2563 -2563 -2257 -2906 -2767 -2767 -2767 -2767 -2767 -2764 -2696 -2696 -3600 -4300 -5330 -5468 -5097 -5467 -4224 -4046 -3620 -3413 -2624 -2624 -1868 -2601 -2601 -2618 -1829 -1950 -949 -593 -679 -1329 -1329 -881 301 289 512 0 0 0 